fe/fe-core/src/test/java/org/apache/doris/analysis/ShowCreateTableStmtTest.java

```diff
@@ -18,46 +18,36 @@
 package org.apache.doris.analysis;
 
 import org.apache.doris.common.AnalysisException;
-import org.apache.doris.mysql.privilege.MockedAuth;
-import org.apache.doris.mysql.privilege.PaloAuth;
-import org.apache.doris.qe.ConnectContext;
-
-import mockit.Mocked;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
-public class ShowCreateTableStmtTest {
-    private Analyzer analyzer;
-
-    @Mocked
-    private PaloAuth auth;
-    @Mocked
-    private ConnectContext ctx;
-
-    @Before
-    public void setUp() {
-        analyzer = AccessTestUtil.fetchAdminAnalyzer(true);
-        MockedAuth.mockedAuth(auth);
-        MockedAuth.mockedConnectContext(ctx, "root", "192.168.1.1");
+import org.apache.doris.common.DdlException;
```

Comment:
Use `control+option+o` remove unused header.
[Doc about it](https://doris.apache.org/zh-CN/developer/developer-guide/java-format-code.html#import-order)

Reply:
Ok, it is still in the development stage, I will deal with it later.